example/chat/resolvers.go

```diff
@@ -9,26 +9,26 @@ import (
 	"time"
 )
 
-type resolver struct {
+type chat struct {
```

Comment:
Maybe `resolver` is too generic a plugin name; this clashed so I changed the resolver struct name here.